### PR TITLE
Extensibility UI - ui-schema version

### PR DESCRIPTION
**Pull request by qbalukom**

### CODE REVIEW

File: core-ui/src/components/Extensibility/components-form/MultiType.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  if (isNil(value)) {
+    selectedType = 'null';
+  } else if (Array.isArray(value)) {
+    selectedType = 'array';
+  } else {
+    selectedType = typeof value;
+  }
+
+  newSchema = newSchema
+    // .set('name', '') // TODO
```
Comment (mrCherry97) [inline core-ui/src/components/Extensibility/components-form/MultiType.js:41]: As we have a talk it should be deleted or make a followup.

File: core-ui/src/components/Extensibility/components-form/index.js [hunk 2]
```diff
@@ -89,6 +92,24 @@ export const widgets = {
...
+  ...widgets,
+  pluginStack: [
+    ReferencingHandler,
+    ExtractStorePlugin,
+    CombiningHandler,
+    DefaultHandler,
+    DependentHandler,
+    ConditionalHandler,
+    // SchemaRulesInjector,
+    // CustomFieldInjector,
```
Comment (mrCherry97) [inline core-ui/src/components/Extensibility/components-form/index.js:109]: To remove

**Review by mrCherry97 — CHANGES_REQUESTED** [review]
- [ ] remove comments,
- [x] Jsonata.js usused `exists`
- [x] path is jsonata in forms
- [ ] in Examples functions data isn't loaded in details
- [ ] image

**Review by mrCherry97 — DISMISSED** [review]

File: docs/extensibility/resources.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ The **general** section is required and contains basic information about the res
   - **group** - API group used for all requests. Not provided for Kubernetes resources in the core (also called legacy) group.
 - **name** - title used in the navigation and on the list screen. It defaults to its resource kind.
 - **category** - the name of a category used for the left-hand menu. By default, it's placed in the `Custom Resources` category.
-- **icon** suffix of an icon name used for the left-hand menu. the default value is `customized`. You can find the list of icons [here](https://sap.github.io/fundamental-react/?path=/docs/component-api-icon--primary).
+- **icon** - suffix of an icon name used for the left-hand menu. the default value is `customized`. You can find the list of icons [here](https://sap.github.io/fundamental-react/?path=/docs/component-api-icon--primary).
```
Comment (IwonaLanger) [inline docs/extensibility/resources.md:37]:
```suggestion
- **icon** - suffix of an icon name used for the left-hand menu. The default value is `customized`. You can find the list of icons [here](https://sap.github.io/fundamental-react/?path=/docs/component-api-icon--primary#icon-list).
```

**Review by mrCherry97 — DISMISSED** [review]

**Review by IwonaLanger — APPROVED** [review]

**Review by IwonaLanger — APPROVED** [review]

**Review by mrCherry97 — APPROVED** [review]

**Pull request merged**